libraries/utils/src/color.ts

```diff
@@ -11,6 +11,31 @@ export function setAlpha(color: DG.ColorType, alpha: number): number {
     DG.Color.b(color);
 }
 
-export function toRgba(color: DG.ColorType): string {
-  return `rgba(${DG.Color.r(color)}, ${DG.Color.g(color)}, ${DG.Color.b(color)}, ${DG.Color.a(color) / 255})`;
+export function intToRgb(value: DG.ColorType): string {
+  return `rgb(${DG.Color.r(value)}, ${DG.Color.g(value)}, ${DG.Color.b(value)})`;
+}
+
+export function intToRgba(value: DG.ColorType): string {
+  return `rgba(${DG.Color.r(value)}, ${DG.Color.g(value)}, ${DG.Color.b(value)}, ${DG.Color.a(value) / 255})`;
+}
+
+export function intToHtml(value: DG.ColorType): string {
+  return `#${(value & 0xFFFFFF).toString(16).padStart(6, '0')}`;
+}
+
+export function intToHtmlA(value: DG.ColorType): string {
+  const alpha = DG.Color.a(value);
+  return `#${(value & 0xFFFFFF).toString(16).padStart(6, '0')}` +
+    alpha.toString(16).padStart(2, '0');
+}
+
+/** Converts color Hex string ('#000bff') to int number */
+export function htmlToInt(value: string): number {
+  return DG.Color.fromHtml(value) & 0xFFFFFF;
+}
+
+/** Converts color Hex string ('#000bff22' , alpha=0x22) to int number with alpha */
+export function htmlToIntA(value: string): number {
+  const m: RegExpExecArray = /#([0-9A-Fa-f]{6})([0-9A-Fa-f]{2})/.exec(value)!;
```

Comment:
comment regular expression

Reply:
/* regexp: dies sign + six hex digits (group 1 for rgb) + two hex digits (group 2 for alpha) */